scipy/sparse/linalg/tests/test_expm_multiply.py

```diff
@@ -63,7 +64,7 @@ def test_expm_multiply(self):
             A = scipy.linalg.inv(np.random.randn(n, n))
             B = np.random.randn(n, k)
             observed = expm_multiply(A, B)
-            expected = np.dot(scipy.linalg.expm(A), B)
+            expected = np.dot(sp_expm(A), B)
```

Comment:
I should explain this somewhere indeed. Previously, we were using sparse implementation in case of a dense input. Here dense gets its own implementation such that sparse one can live in its own environment.

Reply:
Yeah, that then sounds like something that can easily trip up sparse users. `scipy.linalg.expm` no longer being sparse, that is. And we'd prefer it for people to use the top-level version of a name where possible, so `scipy.linalg.expm` was probably the preferred version in the original.